File: app/src/main/java/com/eugene_poroshin/weatherforecast/fragments/TempModePreferenceFragment.java
package com.eugene_poroshin.weatherforecast.fragments;

import android.os.Bundle;

import androidx.preference.PreferenceFragmentCompat;

import com.eugene_poroshin.weatherforecast.R;


public class TempModePreferenceFragment extends PreferenceFragmentCompat {

    @Override
    public void onCreatePreferences(Bundle savedInstanceState, String rootKey) {
        addPreferencesFromResource(R.xml.fragment_preference);
    }

}

File: app/src/main/java/com/eugene_poroshin/weatherforecast/fragments/TempModePreferenceFragment.kt
package com.eugene_poroshin.weatherforecast.fragments

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.eugene_poroshin.weatherforecast.R

class TempModePreferenceFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(
        savedInstanceState: Bundle?,
        rootKey: String?
    ) {
        addPreferencesFromResource(R.xml.fragment_preference)
    }
}